Tests/MapboxCoreNavigationTests/Extensions/LineStringTest.swift

```diff
@@ -0,0 +1,66 @@
+import XCTest
+import Turf
+import CoreLocation
+import TestHelper
+@testable import MapboxCoreNavigation
+
+class LineStringTests: TestCase {
```

Comment:
Should `LineStringTests` be part of Turf instead?

Reply:
Yes, I'll rebase the pr and remove this part after we adopted the latest release of turf-swift that include this method.